app/plugin_install.go

```diff
@@ -91,6 +91,13 @@ func (a *App) installPlugin(pluginFile io.ReadSeeker, replace bool) (*model.Mani
 		return nil, model.NewAppError("uploadPlugin", "app.plugin.store_bundle.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
 
+	// Flag plugin locally as managed by the filestore.
+	if f, err := os.Create(filepath.Join(pluginPath, ".filestore")); err != nil {
```

Comment:
The iddeomatic way to write this (at least a MM) would be:
```go
// Flag plugin locally as managed by the filestore.
f, err := os.Create(filepath.Join(pluginPath, ".filestore"))
if err != nil {
    return nil, model.NewAppError("uploadPlugin", "app.plugin.flag_managed.app_error", nil, err.Error(),http.StatusInternalServerError)
}
f.Close()
```

Reply:
Fixed